CONTRIBUTING.md

```diff
@@ -89,150 +89,176 @@ effect on highlighting. We will work on improving highlighting in the near futur
+@keyword.operator ; operators that are English words (e.g. `and` / `or`)
+@keyword.return   ; keywords like `return` and `yield`
+
+@conditional      ; keywords related to conditionals (e.g. `if` / `else`)
+@repeat           ; keywords related to loops (e.g. `for` / `while`)
+@debug            ; keywords related to debugging [NOTE: unused]
+@label            ; GOTO and other labels (e.g. `label:` in C)
+@include          ; keywords for including modules (e.g. `import` / `from` in Python)
+@exception        ; keywords related to exceptions (e.g. `throw` / `catch`)
 ```
 
-@conceal followed by `(#set! conceal "")` for captures that are not used for highlights but only for concealing.
+#### Types
 
-#### Variables
+```scheme
+@type            ; type or class definitions and annotations
+@type.builtin    ; built-in types
+@type.definition ; type definitions (e.g. `typedef` in C) [NOTE: unused]
+@type.qualifier  ; type qualifiers (e.g. `const`)
 
+@storageclass    ; visibility/life-time/etc. modifiers (e.g. `static`)
+@attribute       ; attribute annotations (e.g. Python decorators)
+@field           ; object and struct fields
+@property        ; similar to `@field`
 ```
-@variable
-@variable.builtin
-@variable.global
+
+#### Identifiers
+
+```scheme
+@variable         ; various variable names
+@variable.builtin ; built-in variable names (e.g. `this`)
+
+@constant          ; constant identifiers
+@constant.builtin  ; built-in constant values
+@constant.macro    ; constants defined by the preprocessor
+
+@namespace        ; modules or namespaces
+@symbol           ; symbols or atoms
 ```
 
 #### Text
 
 Mainly for markup languages.
 
-```
-@text
-@text.strong
-@text.emphasis
-@text.underline
-@text.strike
-@text.title
-@text.literal
-@text.uri
-@text.math (e.g. for LaTeX math environments)
-@text.environment (e.g. for text environments of markup languages)
-@text.environment.name (e.g. for the name/the string indicating the type of text environment)
-@text.reference (for footnotes, text references, citations)
-
-@text.note
-@text.warning
-@text.danger
-
-@todo
+```scheme
+@text                  ; non-structured text
+@text.strong           ; bold text
+@text.emphasis         ; text with emphasis
+@text.underline        ; underlined text
+@text.strike           ; strikethrough text
+@text.title            ; text that is part of a title
+@text.literal          ; literal or verbatim text
+@text.uri              ; URIs (e.g. hyperlinks)
+@text.math             ; math environments (e.g. `$ ... $` in LaTeX)
+@text.environment      ; text environments of markup languages
+@text.environment.name ; text indicating the type of an environment
+@text.reference        ; text references, footnotes, citations, etc.
+
+@text.note             ; info notes
+@text.warning          ; warning notes
+@text.danger           ; danger/error notes
+
+@todo                  ; todo notes [NOTE: unused]
```

Comment:
I'd remove that one, too -- it stands out like a sore thumb in this list.

Reply:
Maybe we could rename it to `@text.todo`?

Reply:
Yes, that would be good (and maybe use it for `@comment` highlight in your follow-up PR).